src/reference/01-General-Info/90-Changes/45-Migrating-from-sbt-0.12.x.md

```diff
@@ -130,3 +139,84 @@ when migrating you mustn't use `.value` but `.evaluated`:
 ```scala
 run := docsRunSetting.evaluated
 ```
+
+### Migrating from the Build trait
+
+With `Build` trait based build such as:
+
+```scala
+import sbt._
+import Keys._
+import xyz.XyzPlugin.autoImport._
+
+object HelloBuild extends Build {
+  val shared = Defaults.defaultSettings ++ xyz.XyzPlugin.projectSettings ++ Seq(
+    organization := "com.example",
+    version      := "0.1.0",
+    scalaVersion := "2.12.1")
+
+  lazy val hello =
+    Project("Hello", file("."),
+      settings = shared ++ Seq(
+        xyzSkipWrite := true)
+    ).aggregate(core)
+
+  lazy val core =
+    Project("hello-core", file("core"),
+      settings = shared ++ Seq(
+        description := "Core interfaces",
+        libraryDependencies ++= scalaXml.value)
+    )
+
+  def scalaXml = Def.setting {
+    scalaBinaryVersion.value match {
+      case "2.10" => Nil
+      case _      => ("org.scala-lang.modules" %% "scala-xml" % "1.0.6") :: Nil
+    }
+  }
+}
+```
+
+You can migrate to `build.sbt`:
+
+```scala
+val shared = Seq(
+  organization := "com.example",
+  version      := "0.1.0",
+  scalaVersion := "2.12.1"
+)
+
+lazy val helloRoot = (project in file("."))
+  .aggregate(core)
+  .enablePlugins(XyzPlugin)
+  .settings(
+    shared,
+    name := "Hello",
+    xyzSkipWrite := true
+  )
+
+lazy val core = (project in file("core"))
+  .enablePlugins(XyzPlugin)
+  .settings(
+    shared,
+    name := "hello-core",
+    description := "Core interfaces",
+    libraryDependencies ++= scalaXml.value
+  )
+
+def scalaXml = Def.setting {
+  scalaBinaryVersion.value match {
+    case "2.10" => Nil
+    case _      => ("org.scala-lang.modules" %% "scala-xml" % "1.0.6") :: Nil
+  }
+}
+```
+
+1. Rename `project/Build.scala` to `build.sbt`.
```

Comment:
The migration path documented here is a migration to an idiomatic `build.sbt` style. In case that's not possible, the Scalajs trick (leaving most in `*.scala`) is a workaround, but I won't recommend it as the first path.